### DISCUSSION TITLE
doerrbohnen

### DISCUSSION THREAD

**Discussion by giscus** [Roald87/roaldin.ch#163] [Announcements]
# doerrbohnen

Dörrbohnen zijn gedroogde sperziebonen1. Om ze te bereiden, moeten de bonen enkele uren worden geweekt. Daarna kunnen ze worden gebakken met ui en kruiden en afgeblust met bouillon. De gedroogde bonen zoals je ze koopt. Adrian Michael, CC-BY-SA-3.0 Ik zag ze vaak in de supermarkt liggen, maar ik kwam nooit in de verleiding om ze te kopen. Tot voor kort had ik ze nog nooit gegeten. Dat veranderde tijdens de middagpauze op het werk. Ze smaakten aardig, maar het zal niet mijn favoriete bereiding van sperziebonen worden. https://de.wikipedia.org/wiki/D%C3%B6rrbohnen ↩

https://roaldin.ch/doerrbohnen

**Comment by oom-wim** [comment]
Dreuge boon'n, zoals ze in Groningen bekend staan, droge witte sperziebonen, andere kleur, zelfde idee. Ik ken een 99-jarige wier lievelingsgerecht dit is. Ze heeft dit jaar nog een stel bonen in de zon gedroogd.
Wat zou de Nederlandse douane ervan maken als je een bundeltje meeneemt?

**Reply by Roald87** [reply]
Ik denk niet dat ze het voor eten aanzien.
